locksmith/scripts/metadataUpload/metadata_upload.ts

```diff
@@ -0,0 +1,126 @@
+import * as sigUtil from 'eth-sig-util'
+import * as ethJsUtil from 'ethereumjs-util'
+import * as Base64 from '../../src/utils/base64'
+
+const args = require('yargs').argv
```

Comment:
Are these things that cannot be imported in the ES6 way? (As in lines 1-3)

Reply:
I don't think so, at least not from my cursory exploration